src/staticdata.c

```diff
@@ -568,6 +568,7 @@ static void jl_serialize_value__(jl_serializer_state *s, jl_value_t *v, int recu
         jl_serialize_value(s, tn->module);
         jl_serialize_value(s, tn->names);
         jl_serialize_value(s, tn->wrapper);
+        jl_serialize_value(s, tn->Typeofwrapper);
         jl_serialize_value_(s, (jl_value_t*)tn->cache, 0);
         jl_serialize_value_(s, (jl_value_t*)tn->linearcache, 0);
```

Comment:
```suggestion
        jl_serialize_value(s, jl_atomic_load_acquire(&tn->Typeofwrapper));
        jl_serialize_value_(s, (jl_value_t*)jl_atomic_load_acquire(&tn->cache), 0);
        jl_serialize_value_(s, (jl_value_t*)jl_atomic_load_acquire(&tn->linearcache), 0);
```

(I don't yet have this file enabled for this static-analysis check, since there are other similar updates missing too)